source/module_hamilt_lcao/module_tddft/test/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+remove_definitions(-D __MPI)
+
+AddTest(
+  TARGET tddft_test
+  LIBS ${math_libs} base device
+  SOURCES tddft_test.cpp upsi_test1.cpp upsi_test2.cpp upsi_test3.cpp propagator_test1.cpp propagator_test2.cpp propagator_test3.cpp middle_hamilt_test.cpp norm_psi_test.cpp bandenergy_test.cpp ../upsi.cpp ../propagator.cpp ../middle_hamilt.cpp ../norm_psi.cpp ../bandenergy.cpp 
```

Comment:
I suggest to add each tests separately with a specific namely to facilitate checking of the corresponding functionalities.